visualization/dashboard/src/MLIDashboard/Cohort.ts

```diff
@@ -7,9 +7,9 @@ export class Cohort {
     private static _cohortIndex: number = 0;
 
     public rowCount: number = 0;
+    public filteredData: Array<{[key: string]: number}>;
```

Comment:
very cool type, "Array<{[key: string]: number}>"!  that's an array of dictionaries in typescript?

Reply:
yes, an array of dictionaries, where the keys are strings and the values are numbers

Reply:
typescript is great, welcome!